### PR TITLE
Fix project save

### PR DESCRIPTION
**Pull request by ClementPasteau**
Do not show in changelog.

The save was not triggered when the project was defined, because the fileMetadata was not initialised anymore.
Also took the opportunity to fix an error appearing when GDJS reloads with recursive-copy, because of latest Node versions (https://github.com/timkendrick/recursive-copy/issues/24)

### CODE REVIEW

File: newIDE/app/package.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
     "prettier": "1.15.3",
     "react-error-overlay": "^6.0.9",
     "react-scripts": "4.0.3",
-    "recursive-copy": "^2.0.10",
```
Comment (AlexandreSi) [inline newIDE/app/package.json:25]: 🧹

**Review by AlexandreSi — APPROVED** [review]
Thanks!

**Review by 4ian — APPROVED** [review]
Make sure to check no errors when creating a new project on the web-app, on the desktop app (that's what you've fixed) and when opening an example from an url on the web-app :)

**Comment by ClementPasteau** [comment]
tested the 3 of them, all look good 👍

**Review by 4ian — APPROVED** [review]

**Pull request merged**